app/routes/circulars._archive._index/PaginationSelectionFooter.tsx

```diff
@@ -0,0 +1,54 @@
+import { useSubmit } from '@remix-run/react'
+import { Select } from '@trussworks/react-uswds'
+
+import CircularPagination from './CircularPagination'
+
+export default function ({
+  page,
+  totalPages,
+  limit,
+  query,
+  formId,
+}: {
+  page: number
+  totalPages: number
+  limit?: number
+  query?: string
+  formId: string
```

Comment:
```suggestion
  form: string
```

to match the name of the prop on `<input>` elements.

Reply:
I am happy to change it. I did it this way for consistency as you originally did it here https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/app/routes/circulars._archive._index/route.tsx#L190 
Would you like me to also change that so that it's consistent across routes? or since the elements the form id is being passed to are different, is there not a need for that level of consistency in variable names?

Reply:
> or since the elements the form id is being passed to are different, is there not a need for that level of consistency in variable names?

That one.